test/test_utils/__init__.py

```diff
@@ -322,7 +322,7 @@ def get_inference_run_command(image_uri, model_names, processor="cpu"):
     if processor == "eia":
         multi_model_location = {
             "resnet-152-eia": "https://s3.amazonaws.com/model-server/model_archive_1.0/resnet-152-eia.mar",
-            "pytorch-densenet": "https://aws-dlc-sample-models.s3.amazonaws.com/pytorch/densenet_eia/densenet_eia.mar",
+            "pytorch-densenet": "https://aws-dlc-sample-models.s3.amazonaws.com/pytorch/densenet_eia/densenet_eia_v1_5_1.mar",
```

Comment:
Replacing this model file will prevent us from being able to test PyTorch 1.3.1 EI DLCs again, right?

Can we have this as a separate entry in this dict, so that older versions of the PT EI DLC can still be tested?

Reply:
Sure, add an entry as below in next revision. 
` "pytorch-densenet-v1-3-1": "https://aws-dlc-sample-models.s3.amazonaws.com/pytorch/densenet_eia/densenet_eia_v1_3_1.mar",
`